assets/src/application/ui/layout/entityList/pagination/EntityPagination.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Pagination } from 'antd';
+import { PaginationProps } from 'antd/lib/pagination';
+
+import './style.scss';
+
+/**
+ * Adds pagination to an "EntityList" component
+ * @return EntityPagination
+ */
+const EntityPagination = ({
+	current,
+	onChange,
+	pageSize = 10,
+	pageSizeOptions = ['10', '20', '30', '40'],
+	showTotal,
+	total,
+}: PaginationProps) => {
+	return (
+		<div className='ee-entity-pagination'>
+			<Pagination
+				current={current}
+				onChange={onChange}
+				pageSize={pageSize}
+				pageSizeOptions={pageSizeOptions}
+				showTotal={showTotal}
+				total={total}
+			/>
```

Comment:
should add a `{...rest}` prop to this so that we can utilize other props from here: https://ant.design/components/pagination/#API

Reply:
The initial thought was to have more visibility/ control on what we need and do pass to pagination and add as needed, not a biggie; updated as required: https://github.com/eventespresso/event-espresso-core/pull/2372/commits/98dad297cd2b67e4ff76222a0a0ad7c319abb532